docs/advanced-usage/access-control/kerberos/index.md

```diff
@@ -1 +1,62 @@
 # Kerberos
+
+If your workflow requires Kerberos authentication, you should set `kerberos: true`
+for the steps in need in the workflow specification. Please note that step's
+docker image (e.g ``environment: 'cern/slc6-base'``) should have Kerberos client
+installed and you have to [upload keytab file and username](../../compute-backends/htcondor/index.md#Authentication)  for the Kerberos authentication to work.
+
+Serial example:
```

Comment:
Since this page is now "far" from workflow pages, this could be better said:

```
Example for the Serial workflow:
```

(ditto for CWL and Yadage)

We could also cross-link into Serial, CWL, Yadage pages.

Note: due to shortage of time, we can go into express-merging mode, i.e. not paying attention into these cosmetic/stylistic language comments, and later on, after the Workshop, the "editor" can go through the docs added by various "authors" and amend the style to unify/harmonise the docs. WDYT?